Bundle/Resources/config/message_converters.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\DependencyInjection\Loader\Configurator;
+
+use Symfony\Bridge\PsrHttpMessage\ArgumentValueResolver\PsrServerRequestResolver;
+use Symfony\Bridge\PsrHttpMessage\EventListener\PsrResponseListener;
+use Symfony\Component\DependencyInjection\Reference;
+
+return static function (ContainerConfigurator $container) {
+    $container->services()
+        ->set('psr_http_message.response_listener', PsrResponseListener::class)
+            ->args([new Reference('psr_http_message.http_foundation_factory')])
+            ->tag('kernel.event_subscriber')
+
+        ->set('psr_http_message.server_request_resolver', PsrServerRequestResolver::class)
+            ->args([new Reference('psr_http_message.psr_http_factory')])
+            ->tag('controller.argument_value_resolver')
+    ;
```

Comment:
Is it possible to enable these automatically (and then maybe allow them to be DISABALED)? I'm trying to avoid users almost ever needing the config file. The first could always be enabled (the service is available, is there a downside)? The second could be enabled if/once `psr_http_message.psr_http_factory` is present. I'm trying to give the user more, with less config. WDYT?

Reply:
The downside is runtime overhead on the hotpath yes.

Reply:
Ok, let's not do that then ;)